src/reporter.ts

```diff
@@ -134,16 +149,20 @@ class RPReporter implements Reporter {
       };
       const suiteObj = this.client.startTestItem(startSuiteObj, this.launchId);
       this.addRequestToPromisesQueue(suiteObj.promise, 'Failed to start suite.');
-      this.suites.set(suiteTitle, {
+      this.suitesStackTempId.push(suiteObj.tempId);
+      this.suites.set(suiteObj.tempId, {
         id: suiteObj.tempId,
         name: suiteTitle,
+        ...(this.suitesAttributes.length && {
+          attributes: this.suitesAttributes[this.suitesStackTempId.length - 1],
+        }),
```

Comment:
If you set suite attributes just here, may not necessary to send them on finish? According to your implementation them can be added to the suite on start request.